tempo_project/README.md

```diff
@@ -117,6 +117,41 @@ Each dataset folder contains:
 
 ### Working with Datasets
 
+Data can be reached at:
+    - /datasets/dsgrid/tempo-v2022 on kestrel
+    - s3://nrel-pds-dsgrid/tempo/tempo-2022/v1.0.0 on [OEDI](https://data.openei.org/s3_viewer?bucket=nrel-pds-dsgrid&prefix=tempo%2Ftempo-2022%2Fv1.0.0%2F)
+
+There are many ways to read the files in this dataset, all datasets provide [parquet files](https://parquet.apache.org/) and some smaller datasets like `annual_summary_state` provide csv's in addition. 
+
+#### Loading with duckdb
+
+Duckdb can be [installed](https://duckdb.org/docs/installation/) on Mac, Windows, or Linux to provide an interface on terminal/command prompt. The duckdb interface uses SQL language to read from parquet files. 
+
+Example loading state level data from Colorado for efs_high_ldv scenario with a duckdb query from s3:
```

Comment:
This may be missing a couple of details. You should probably show how to enter this shell in the terminal.
```bash
$ duckdb
```
and then describe how you can optionally choose a filename in which to store data that is downloaded in the subsequent commands.